packages/xo-web/src/xo-app/logs/backup-ng/log-alert-header.js

```diff
@@ -31,7 +31,7 @@ export default decorate([
     effects: {
       _downloadLog: () => ({ formattedLog }, { log }) =>
         downloadLog({ log: formattedLog, date: log.start, type: 'backup NG' }),
-      restartFailedVms: () => async (
+      restartFailedVms: (_, { force }) => async (
```

Comment:
The argument may be `undefined` which will trigger an error.
https://github.com/vatesfr/xen-orchestra/blob/fc5714336ed8cdfd224436d05826339294eccd95/packages/xo-web/src/common/action-button.js#L81-L83

Reply:
why the argument would be undefined ? 
https://github.com/vatesfr/xen-orchestra/blob/fc5714336ed8cdfd224436d05826339294eccd95/packages/xo-web/src/common/action-button.js#L75-L84
In my case, `empty` will always be false